module/VuFind/src/VuFind/Db/Entity/LoginToken.php

```diff
@@ -0,0 +1,357 @@
+     *          type="datetime",
+     *          nullable=false
+     * )
+     */
+    protected $lastLogin = '2000-01-01 00:00:00';
+
+    /**
+     * Browser.
+     *
+     * @var ?string
+     *
+     * @ORM\Column(name="browser",
+     *             type="string",
+     *             length=255,
+     *             nullable=true
+     * )
+     */
+    protected $browser;
+
+    /**
+     * Platform.
+     *
+     * @var ?string
+     *
+     * @ORM\Column(name="platform",
+     *             type="string",
+     *             length=255,
+     *             nullable=true
+     */
+    protected $platform;
+
+    /**
+     * Expires.
+     *
+     * @var int
+     *
+     * @ORM\Column(name="expires",
+     *             type="integer",
+     *             nullable=false
+     * )
+     */
+    protected $expires;
+
+    /**
+     * Last session ID.
+     *
+     * @var ?string
+     *
+     * @ORM\Column(name="last_session_id",
+     *             type="string",
+     *             length=255,
+     *             nullable=true
+     */
+    protected $lastSessionId;
+
+    /**
+     * Getter for ID.
+     *
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Setter for User.
+     *
+     * @param UserEntityInterface $user User to set
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setUser(UserEntityInterface $user): LoginTokenEntityInterface
+    {
+        $this->user = $user;
+        return $this;
+    }
+
+    /**
+     * User getter (only null if entity has not been populated yet).
+     *
+     * @return ?UserEntityInterface
+     */
+    public function getUser(): ?UserEntityInterface
+    {
+        return $this->user;
+    }
+
+    /**
+     * Set token string.
+     *
+     * @param string $token Token
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setToken(string $token): LoginTokenEntityInterface
+    {
+        $this->token = $token;
+        return $this;
+    }
+
+    /**
+     * Get token string.
+     *
+     * @return string
+     */
+    public function getToken(): string
+    {
+        return $this->token;
+    }
+
+    /**
+     * Set series string.
+     *
+     * @param string $series Series
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setSeries(string $series): LoginTokenEntityInterface
+    {
+        $this->series = $series;
+        return $this;
+    }
+
+    /**
+     * Get series string.
+     *
+     * @return string
+     */
+    public function getSeries(): string
+    {
+        return $this->series;
+    }
+
+    /**
+     * Set last login date/time.
+     *
+     * @param DateTime $dateTime Last login date/time
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setLastLogin(DateTime $dateTime): LoginTokenEntityInterface
+    {
+        $this->lastLogin = $dateTime->format('Y-m-d H:i:s');
```

Comment:
This can just be `= $dateTime;` -- no need to format, since we're storing the object instead of a string.